airflow-core/src/airflow/configuration.py

```diff
@@ -860,7 +860,7 @@ def _create_future_warning(name: str, section: str, current_value: Any, new_valu
         )
 
     def mask_secrets(self):
-        from airflow.sdk.execution_time.secrets_masker import mask_secret
+        from airflow._shared.secrets_masker.secrets_masker import mask_secret
```

Comment:
I wonder if this should be a case where we try to mask in both maskers too?

Reply:
Yeah makes sense. Even though its a module level singleton, the module is different for python